software/contrib/consequencer.py

```diff
@@ -123,12 +134,8 @@ def clockTrigger():
                 else:
                     cv3.value(int(self.HH[self.pattern][self.step]))
 
-            # Reset clock step at 128 to avoid a HUGE integer if running for a long time
-            # over a really long period of time this would look like a memory leak
-            if self.clock_step < 128:
-                self.clock_step +=1
-            else:
-                self.clock_step = 0
```

Comment:
I added it to avoid a growing memory leak as Python just keeps adding more and more memory space to integers that keep growing. However, after testing for a long time I don't believe I need to worry about a memory leak.
Additionally, this simple if statement added latency that I want to keep to minimum.